site/css/style.css

```
@@ -1135,109 +1467,1199 @@ section { padding: 5rem 0; }
+  gap: 0.8rem;
+  flex-wrap: wrap;
+}
+.wizard-resume-banner button {
+  background: transparent;
+  border: 1px solid currentColor;
+  border-radius: 999px;
+  color: inherit;
+  padding: 0.25rem 0.7rem;
+  font-size: 0.78rem;
+  cursor: pointer;
+  font: inherit;
```

Stylelint (17.10.0)

[error] Overridden property "font-size" by shorthand "font" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)